pyvista/plotting/tools.py

```diff
@@ -333,3 +343,43 @@ def opacity_transfer_function(mapping, n_colors, interpolate=True,
             raise RuntimeError(f'Transfer function cannot have more values than `n_colors`. This has {mapping.size} elements')
         return mapping
     raise TypeError(f'Transfer function type ({type(mapping)}) not understood')
+
+
+def parse_color(color, opacity=None, default_color=None):
+    """Parse color into a vtk friendly rgb list.
+
+    Values returned will be between 0 and 1.
+
+    """
+    if color is None:
+        if default_color is None:
+            color = pyvista.global_theme.color
+        else:
+            color = default_color
+    if isinstance(color, str):
+        color = string_to_rgb(color)
+    elif len(color) == 3:
+        pass
+    elif len(color) == 4:
+        color = color[:3]
+    else:
+        raise ValueError(f"""
+    Invalid color input: ({color})
+    Must be string, rgb list, or hex color string.  For example:
+        color='white'
+        color='w'
+        color=[1, 1, 1]
+        color='#FFFFFF'""")
+    if opacity is not None and isinstance(opacity, (float, int)):
+        color = [color[0], color[1], color[2], opacity]
+    return color
+
+
+def parse_font_family(font_family):
+    """Check font name."""
+    font_family = font_family.lower()
+    if font_family not in FONTS._member_names_:
```

Comment:
`_member_names_` seems undocumented, looking at https://docs.python.org/3/library/enum.html. Perhaps we should use either `__members__` or `[font.name for font in FONTS]`?

Reply:
Agreed.  Anyway dunder methods are implementation rather than API, so it was a mistake to add it here.